### ISSUE TITLE
In contenttypes the 'name' doesn't get generated from slug anymore.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#5961]
Details
-------

 - Relevant Bolt Version: [ 3.1 | 3.2 | master ]

Reproduction
------------

Add a contenttype like this: 

```
blocks:
    slug: blocks
    singular_name: Block
    tablename: blocks
    fields:
        title:
            type: text
            class: large
            group: "Block"
        slug:
            type: slug
            uses: [ title ]
        content:
            type: html
            height: 150px
```

Note the `slug:` instead of the `name:`. 

Go to the bolt backend, get an error: 

![screen shot 2016-10-26 at 16 05 16](https://cloud.githubusercontent.com/assets/1833361/19729076/047fb0a2-9b96-11e6-9d85-ee154030b36e.png)

**Comment by bobdenotter** [comment]
By the way: Same thing happens if you use `singular_slug: block` instead of `singular_name: block`

**Comment by bobdenotter** [comment]
Fixed in #5962
